tests/file.go

```
@@ -203,7 +203,7 @@ func RunFileTest(rootDir string, path string, nativeLibs bool, logger loggerFunc
 				// errWanted given
 				if errWanted != "" {
 					if pnc == nil {
-						panic(fmt.Sprintf("fail on %s: got nil error, want: %q", path, errWanted))
+						return fmt.Errorf("fail on %s: got nil error, want: %q", path, errWanted)
```

I found better to return an error here, so the tests are not interrupted.

---

Actually break some other tests, so I reverted those.